.github/workflows/build-test.yml

```diff
@@ -0,0 +1,24 @@
+name: build-and-test
+
+on: [push]
+
+jobs:
+
+  build-and-test:
+    name: Build and test
+    runs-on: ubuntu-latest
+    container: domainr/ci:swan
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Install dependencies
+        run: go get -t -d -v ./...
```

Comment:
This will update the go.mod file. We should prefer `go mod download` or just omitting this step altogether and letting `go build` and/or `go test` install the dependencies.